docs/entity-cheat-sheet.rst

```diff
@@ -0,0 +1,333 @@
+.. spelling:word-list::
+    hh
+    num
+
+.. comment
+   add an og image like this :og:image: https://docs.getodk.org/_static/img/tutorial-first-form.png
+
+Entity Cheat Sheet
+=================================
+
+This page provides a quick reference for how to design forms in :doc:`XLSForm </tutorial-first-form/>` for ODK that use, create, and update Entities.
+
+Creating and Updating Entity Data
+---------------------------------
+
+When you are designing your form, you can specify how transform submissions into entities. (Learn more about entities :doc:`here </entities-intro/>`.)
+
+To make a form that creates or updates entities, there are two places to make changes in your form: the ``survey`` sheet and the ``entities`` sheet.
```

Comment:
Let's capitalize Entities when we mean the ODK Entity concept. I'm not super sold on capitalizing Form and Submission because those have the same meaning within and outside of ODK but we talk about both entity and Entity.